scripts/pre-install.sh

```diff
@@ -1,10 +1,16 @@
-#!/bin/sh
+#!/bin/bash
 
-# Copy existing configuration if pre-existing installation is found
-test -f /etc/opt/influxdb/influxdb.conf
-if [ $? -eq 0 ]; then
-    mkdir /etc/influxdb
-    cp -a /etc/opt/influxdb/* /etc/influxdb/
-fi
+if [[ -d /etc/opt/influxdb ]]; then
+    # Legacy configuration found
+    if [[ ! -d /etc/influxdb ]]; then
+	# New configuration does not exist, move legacy configuration to new location
+	echo -e "Please note, InfluxDB's configuration is now located at '/etc/influxdb' (previously '/etc/opt/influxdb')."
+	mv -vn /etc/opt/influxdb /etc/influxdb
 
-exit
+	if [[ -f /etc/influxdb/influxdb.conf ]]; then
```

Comment:
I'd be happy to demo this, btw. I tried all sorts of methods to prevent it, but rpm overwrites the configuration every-friggin-time.